framework/resources/Functional/s3/files/files.json

```diff
@@ -0,0 +1,23 @@
+{
+  "testId": "files_table_for_s3",
+  "type": "group",
+  "description": "Querying FILES table in the INFORMATION_SCHEMA for the s3",
+  "submit-type": "jdbc",
+  "categories": [
+    "functional"
+  ],
+  "dependencies" : ["s3"],
+  "matrices": [
+    {
+      "query-file": ".*.sql",
+      "schema": "s3.tmp",
+      "output-format": "tsv",
+      "expected-file": ".*.e_tsv",
+      "verification-type": [
+        "in-memory"
+      ]
+    }
+  ],
+  "datasources": []
```

Comment:
Does this need no cp/gen sections? We just want to make sure any test suite runs independently..

Reply:
Minio server runs on the local folder. I placed test data to the **framework/resources/Datasources/s3/minio/minio_data/tstbucket/tmp/** and now I don't need to copy any files to the other place. Minio will get access to these files via the S3 API.